cla_public/apps/base/__init__.py

```diff
@@ -22,3 +22,12 @@ def remember_locale(response):
             response = redirect(path)
             response.set_cookie("locale", locale, expires=expires)
             return response
+
+
+@base.before_app_request
+def detect_maintenance():
+    maintenance_mode = current_app.config.get("MAINTENANCE_MODE", False)
+    if maintenance_mode and request.path != u"/maintenance":
+        return redirect("/maintenance")
+    if not maintenance_mode and request.path == u"/maintenance":
+        return redirect("/start")
```

Comment:
I wonder if this should be `/` so that prod goes to the external service start page